dynatrace_extension/cli/main.py

```diff
@@ -40,6 +40,7 @@ def version():
 def run(
     extension_dir: Path = typer.Argument("."),
     activation_config: str = "activation.json",
+    secrets: str = "secrets.json",
```

Comment:
@tomas-s-roos this needs to be an optional argument so we don't break contract

Reply:
Here this is already optional, or am I missing something? It has a default value.